modules/kafka/kafka_test.go

```diff
@@ -87,41 +81,3 @@ func TestKafka(t *testing.T) {
 		t.Fatalf("expected value to be %s, got %s", "value", string(consumer.message.Value))
 	}
 }
-
-func TestKafka_invalidVersion(t *testing.T) {
-	ctx := context.Background()
-
-	_, err := kafka.Run(ctx, "confluentinc/confluent-local:6.3.3", kafka.WithClusterID("kraftCluster"))
-	if err == nil {
-		t.Fatal(err)
-	}
-}
-
-// assertAdvertisedListeners checks that the advertised listeners are set correctly:
-// - The BROKER:// protocol is using the hostname of the Kafka container
-func assertAdvertisedListeners(t *testing.T, container testcontainers.Container) {
-	inspect, err := container.Inspect(context.Background())
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	hostname := inspect.Config.Hostname
-
-	code, r, err := container.Exec(context.Background(), []string{"cat", "/usr/sbin/testcontainers_start.sh"})
```

Comment:
this script doesn't exist anymore in the kafka image - also I didn't think it was necessary to check advertisted listeners, considering if it's set incorrectly, the other tests would fail because kafka wouldn't work in the first place